mobile/src/components/uam/users-table/users-table.tsx

```diff
@@ -37,7 +37,7 @@ const UsersTable: FC = () => {
         count: PaginationDefaultValue.DEFAULT_COUNT,
       }),
     );
-  }, [page, users]);
+  }, [page, userDeleteDataStatus]);
```

Comment:
Do we need userDeleteDataStatus as dependency here?

Reply:
Yes, to upadate users table after delete

Reply:
we should update it in reducer in deleteUser action

Reply:
![Screenshot_1](https://user-images.githubusercontent.com/89215185/185904879-960bd0db-ad5e-409a-81a1-a112abbb0341.png)
we've got items.length that provides a bug, because we've changed total count to 4 after deleting 1 user.(it broke the pagination)
if I made the screenshot changes in reducer - it won't break the pagination, but we left only 4 users in the table after deleting
when we provide the dependency of deleteUserStatus - we refresh or table and got 1 new person in it

Reply:
@revenets please take a look to frontend implementation. We can also trigger useEffect of getUsers by totalCount changes